data/json/items/archery.json

```diff
@@ -840,7 +840,7 @@
       "price" : 324000, "//" : "Modern crossbows can come in anywhere from $300 to $2K.  Large plus Antique = expensive.",
       "material" : ["iron", "wood"],
       "flags" : ["FIRE_TWOHAND", "STR_RELOAD", "PRIMITIVE_RANGED_WEAPON", "TRADER_AVOID"],
-      "skill" : "archery",
+      "skill" : "rifle",
```

Comment:
The aspects I'm thinking of are bulk, weight and inaccuracy, Gantz has all of thse in spades, it's essentially a siege engine you can barely carry.

Basically if no one has an argument for switching it to launcher I'll leave it using the rifle skill :)

Honestly the minigun probably shouldn't use rifle either, but I don't think launcher is any better, nothing really fits at all.